scss/mixins/_buttons.scss

```diff
@@ -63,17 +63,15 @@
   background-image: none;
   background-color: transparent;
   border-color: $color;
-
-  &:focus,
-  &.focus,
-  &:active,
-  &.active,
-  .open > &.dropdown-toggle {
+  
+  @include hover-focus-active {
     color: #fff;
     background-color: $color;
-        border-color: $color;
+        border-color: $color;  
```

Comment:
Line should be indented 4 spaces, but was indented 8 spaces<br>Line contains trailing whitespace